### PR TITLE
Update calendar summary

### PR DESCRIPTION
**Pull request by jdrieghe**
### Changed
- Updated the dependency on `cultuurnet/calendar-summary-v3` to `3.x`

### Removed
- implicit dependency on `cultuurnet/search-v3`
- some unnecessary abstractions around Sapi3 search query factories

---
Ticket: https://jira.uitdatabank.be/browse/III-3830

### CODE REVIEW

File: src/Offer/Security/SearchQueryFactory.php [hunk 1]
```diff
@@ -1,26 +0,0 @@
...
-declare(strict_types=1);
-
-namespace CultuurNet\UDB3\Offer\Security;
-
-use ValueObjects\StringLiteral\StringLiteral;
-
-/**
- * Implementation of the search query factory for SAPI3.
- */
-class SearchQueryFactory extends SearchQueryFactoryBase
```
Comment (bertramakers) [inline src/Offer/Security/SearchQueryFactory.php:12]: To be sure I understand correctly, this was unrelated but you spotted this by coincidence? Nice find! 🕵️

**Review by bertramakers — COMMENTED** [review]
Love it! 👍

**Review by bertramakers — APPROVED** [review]

File: src/Offer/Security/SearchQueryFactory.php (hunk 1 above)
Comment (jdrieghe) [reply src/Offer/Security/SearchQueryFactory.php:12]: Yeah, it had dependencies on the Query VO but it wasn't used at all

**Comment by LucWollants** [comment]
Some nice boy scouting, that deserves a 🏅

**Pull request merged**